test/main-hook-tests.mjs

```diff
@@ -186,13 +184,13 @@ describe("main hook", function () {
       flags.push("--preserve-symlinks")
     }
 
-    flags.forEach((flag) => {
+    flags.forEach((flag) =>
```

Comment:
Brackets are ok for `forEach` use. It reminds me that the return value is not significant or implied.

Reply:
ah, ok. I'll revert.